salt/cloud/clouds/saltify.py

```diff
@@ -47,30 +54,150 @@ def __virtual__():
     return True
 
 
-def list_nodes():
+def _get_connection_info():
     '''
-    Because this module is not specific to any cloud providers, there will be
-    no nodes to list.
+    Return connection information for the passed VM data
     '''
-    return {}
+    vm_ = get_configured_provider()
+
+    try:
+        ret = {'username': vm_['username'],
+               'password': vm_['password'],
+               'eauth': vm_['eauth'],
+               'vm': vm_,
+               }
+    except IndexError:
+        raise SaltCloudException(
+            'Configuration must define salt-api "username", "password" and "eauth"')
+    return ret
 
 
-def list_nodes_full():
+def avail_locations(call=None):
     '''
-    Because this module is not specific to any cloud providers, there will be
-    no nodes to list.
+    This function returns a list of locations available.
+    salt-cloud --list-locations my-cloud-provider
```

Comment:
There are several references that look this way in the functions below as well. Adding these docs are great, but providing the actual command examples formatted with `.. code-block:: bash` would be stellar.